addons/mail/static/src/js/mail.js

```diff
@@ -1060,11 +1074,16 @@ var ComposeMessage = Attachment.extend ({
     },
 
     start: function () {
+        var self = this;
         this.ds_attachment = new data.DataSetSearch(this, 'ir.attachment');
         this.fileupload_id = _.uniqueId('oe_fileupload_temp');
         $(window).on(this.fileupload_id, this.on_attachment_loaded);
         this.$(".o_timeline_msg_attachment_list").html(this.display_attachments(this));
         this.bind_events();
+        // Store result: current user is from employee group ? (For Mention widget)
+        new Model('res.users').call('has_group', ['base.group_user']).done(function(is_employee) {
```

Comment:
Are you sure this is required ? can portal user read and search on res.partner ? If not, remove this rpc call.